Change when to initial setup

reoono opened the pull request:
## References
Fixes #129, #76

## Code changes
Changed when to initial setup.

I wrote details in #129, but the summary of the reason for the change are shown below.

### Summary
The  bug of #129 is caused by the following reasons

    When JupyterLab starts up, `jupyter-fs's refresh()` being executed before `regular-table's connectedCallback()` is executed.
    However, that does not complete the necessary initialization and generates a null reference.

After `application's start()` is completed, `connectedCallback()` has already been executed.

Therefore, I changed it so that `refresh()` is executed at that timing.

## User-facing changes
None

## Backwards-incompatible changes
None

StefanBrand commented:
There is an error message in [the JS lint job](https://dev.azure.com/tpaine154/jupyter/_build/results?buildId=2875&view=logs&j=3fee6a13-5df0-52df-93ca-d3a8fed43d64&t=0472864a-d307-5525-e6e2-1270f5ed84d8):

>   136:5  error    Promises must be handled appropriately or explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

I'm not sure about the `Binder Badge / binder (pull_request_target)` job.

The Mac Python37 job says this:

> ##[warning]An image label with the label macos-10.14 does not exist.
> ,##[error]The remote provider was unable to process the request.
___

Apart from that, does this really address the OP of #76? They specifically say:

> Nothing new appears.

In #129 _something_ new appears, but it has no content. Maybe I'm being to pedantic on the wording, though. :smile:

reoono commented:
Thank you for your prompt comment!

## lint error
First, I fixed the lint error that originated from the code I wrote.

When `started` does not resolve(), i.e., application.start() fails, JupyterLab cannot be started 
so I think that error handling by catch is unnecessary and added void operator.

## other errors 
Other errors are not directly related to this fix and will be addressed in a separate PR.

(I think that the issue was created in #121.
For reference, I believe following is the cause of the Mac Python37 job error.
https://devblogs.microsoft.com/devops/hosted-pipelines-image-deprecation/）

## issue 76
Thank you for pointing that out, I had forgotten that you created a separate issue for that reason.
Indeed, we should not assume that it is directly related to #76, until the author( @JohnOmernik ) comments.
(In other words, even if this PR is merged, #76 should not be closed.)
